tests/test_FFBbasis2D.py

```diff
@@ -17,49 +16,46 @@
 logger = logging.getLogger(__name__)
 DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
 
+params = [
```

Comment:
Maybe we should import these from one place instead of copying them.

It might be simplest to have a `basis_params_2d` and `basis_params_3d` that live in `_basis_utils` and are just imported into FB2D, FFB2D, FB3D and FFB3D. Perhaps I overlooked something that would block this.

Reply:
Moved them into `_basis_util`

Reply:
Great thanks.